ansible_runner/__main__.py

```diff
@@ -217,6 +217,24 @@ def main(sys_args=None):
     parser.add_argument("-a", "--args", dest='module_args',
                         help="Module arguments")
 
+    parser.add_argument("--process-isolation", dest='process_isolation', default=False,
```

Comment:
one last thing that I hadn't though of before now... I wonder if we should switch this to `store_true` so that we don't end up requiring a positional argument after the `--process_isolation` flag.

Basically, passing `--process_isolation` on its own leads to PI mode being enabled rather than needing set pass the value T/F.

Reply:
Yeah, I agree with that. That's a better UX.